docs/vexim-acl-check-rcpt.conf

```diff
@@ -1,3 +1,18 @@
+# spamtrap write sender IP to BL file when receiving messages to special
+# adresses defined in the file 'CONFDIR/spamtrap_receiver'
+deny
+  message = ${run{/bin/bash -c "/bin/echo $sender_host_address >> /etc/exim4/local_host_blacklist;"}}
```

Comment:
And where is `/etc/exim4/local_host_blacklist` used? If you block mails from hosts in `/etc/exim4/local_host_blacklist` you have to consider that only an account was hijacked (gmail, yahoo, ...) and also a grace period after which you remove the host.